test/functional/Java9andUp/src/org/openj9/test/java/lang/Test_Fma.java

```diff
@@ -0,0 +1,549 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2019 IBM Corp. and others
```

Comment:
I'm not sure if there is a file naming scheme in place for these tests, but I would prefer `Test_Math_fma.java` for this file and `Test_StrictMath_fma.java` for the other.  cc @smlambert in case she has some guidance to offer.

Reply:
Fixed in 1907302